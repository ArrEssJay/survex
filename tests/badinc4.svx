; The two lines below used to cause a segfault (fixed in 1.2.7)
*include ; comment
*include
*fix 1 reference 0 0 0
