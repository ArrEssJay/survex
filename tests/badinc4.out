./badinc4.svx:2: Expecting string field
./badinc4.svx:3: Expecting string field

Removing trailing traverses...

Concatenating traverses between nodes...

Simplifying network...

Calculating network...

Calculating traverses between nodes...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
Done.
There were 0 warning(s) and 2 error(s) - no output files produced.
